services/director-v2/src/simcore_service_director_v2/models/domains/comp_tasks.py

```diff
@@ -78,23 +103,23 @@ class Config:
 class CompTaskAtDB(BaseModel):
     project_id: ProjectID
     node_id: NodeID
-    job_id: Optional[str] = Field(None, description="The worker job ID")
+    job_id: Optional[str] = Field(default=None, description="The worker job ID")
     node_schema: NodeSchema = Field(..., alias="schema")
     inputs: Optional[InputsDict] = Field(..., description="the inputs payload")
     outputs: Optional[OutputsDict] = Field({}, description="the outputs payload")
     run_hash: Optional[str] = Field(
-        None,
+        default=None,
         description="the hex digest of the resolved inputs +outputs hash at the time when the last outputs were generated",
     )
     image: Image
     submit: datetime
-    start: Optional[datetime]
-    end: Optional[datetime]
+    start: Optional[datetime] = Field(default=None)
+    end: Optional[datetime] = Field(default=None)
     state: RunningState
-    task_id: Optional[PositiveInt]
+    task_id: Optional[PositiveInt] = Field(default=None)
     internal_id: PositiveInt
     node_class: NodeClass
-    errors: Optional[list[ErrorDict]]
+    errors: Optional[list[ErrorDict]] = Field(default=None)
```

Comment:
Why add all these default None values? I guess it does not have an impact on the current codebase

Reply:
It has an impact with the static type check system
We mentioned some PRs ago that pylance will complain if the defaults where not explicitly set as ``Field(default=** )``

Here [it defaults to None](https://github.com/ITISFoundation/osparc-simcore/blob/bee16a3398a8ee5ed6fe99f5a17cbbfaec7d75f7/packages/settings-library/tests/test__pydantic_settings.py#L109) and when we create an instance in [comp_task.py](https://github.com/ITISFoundation/osparc-simcore/blob/dab37fd0c94e56679f2ce1d9d97f61cd538df09f/services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py#L87) it was highlighted as error